src/current/v25.2/row-level-security.md

```diff
@@ -0,0 +1,561 @@
+toc: true
+keywords: security, row level security, RLS
+docs_area: develop
+---
+
+Row Level Security (_RLS_) is a security feature that allows organizations to restrict access to specific rows of data in a database based on user [roles]({% link {{ page.version.version }}/security-reference/authorization.md %}#roles), [permissions]({% link {{ page.version.version }}/security-reference/authorization.md %}#authorization-models), or other criteria.
+
+Row-level security complements standard SQL privileges ([`GRANT`]({% link {{ page.version.version }}/grant.md %})/[`REVOKE`]({% link {{ page.version.version }}/revoke.md %})) by allowing administrators to define policies that determine precisely which rows users can view or modify within a specific table.
+
+## Use cases
+
+Use cases for row-level security include:
+
+- [Restricting access to sensitive data for compliance](#restricting-access-to-sensitive-data-for-compliance)
+- [Designing multi-tenant applications](#designing-multi-tenant-applications)
+
+### Restricting access to sensitive data for compliance
+
+In industries like finance or healthcare, organizations are required to ensure that only authorized users access sensitive data. Row-level security (RLS) addresses this requirement directly within the database.
+
+For example, RLS allows a financial institution to restrict access to customer records based on roles or departments. In healthcare, RLS can be used to enforce policies ensuring patient records are visible only to the medical staff involved in their care.
+
+RLS embeds access control logic directly into the database and eliminates the need for manual filtering in application code. This centralized enforcement prevents inconsistencies, reduces security attack surface, and simplifies compliance with data access regulations.
+
+For an example, see [RLS for Data Security (Fine-Grained Access Control)](#rls-for-data-security-fine-grained-access-control).
+
+### Designing multi-tenant applications
+
+In multi-tenant applications such as typical Software-as-a-Service (SaaS) deployments, isolating data between tenants within shared tables is a requirement. Row-Level Security (RLS) provides a database-level mechanism for enforcing this isolation. SaaS providers can utilize RLS policies to ensure tenants can only access their own data, eliminating the need for complex and potentially insecure application-layer filtering logic based on tenant IDs.
+
+For an example, see [RLS for Multi-Tenant Isolation](#rls-for-multi-tenant-isolation).
+
+## How to use row-level security
+
+At a high level, the steps for using row-level security (RLS) are as follows:
+
+1. Create [schema objects]({% link {{ page.version.version }}/schema-design-overview.md %}) and [insert data]({% link {{ page.version.version }}/insert-data.md %}). ([`CREATE TABLE`]({% link {{ page.version.version }}/create-table.md %}), [`INSERT`]({% link {{ page.version.version }}/insert.md %}))
```

Comment:
nit: RLS may be enabled on existing tables. This may seem obvious, but I thought I'd mention it because you could infer here that it can only be set on new tables.